api/python/cellxgene_census/tests/experimental/pp/test_online.py

```diff
@@ -3,7 +3,7 @@
 import pytest
```

Comment:
Can you add some test cases for the expected error conditions?  For example, using MeanAccumulator with n_samples==0 or n_variables==0